Berksfile

```diff
@@ -4,8 +4,8 @@ source 'https://supermarket.chef.io'
 metadata
 
 cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
-cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
-cookbook 'hops', github: "logicalclocks/hops-hadoop-chef", branch: "master"
+cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "karamel_installer"
+cookbook 'hops', github: "logicalclocks/hops-hadoop-chef", branch: "karamel_installer"
```

Comment:
Kind reminder to set back to `master` ;)

Reply:
ok